Storage/tests/System/BucketRpoTest.php

```diff
@@ -63,13 +63,13 @@ public function testUpdateRpo()
     public function testGetRpoOnNonDualRegionBucket()
     {
         // self::$bucket is already created in StorageTestCase::setUp
-        $this->assertNull(self::$bucket->info()['rpo']);
+        $this->assertArrayNotHasKey('rpo', self::$bucket->info());
```

Comment:
@bshaffer / @danielbankhead `rpo` metadata isn't returned & test is failing in `main` branch too. 
The validation in test still remains the same!